vehicle/OVMS.V3/components/vehicle_mgev/src/mg_poll_vcu.cpp

```diff
@@ -76,8 +76,16 @@ void OvmsVehicleMgEv::IncomingVcuPoll(
             );
             break;
         case vcuChargerConnectedPid:
-            StandardMetrics.ms_v_door_chargeport->SetValue(data[0] & 1);
-            StandardMetrics.ms_v_charge_pilot->SetValue(data[0] & 1);
+            if (StandardMetrics.ms_v_charge_type->AsString() == "ccs")
+            {
+                StandardMetrics.ms_v_door_chargeport->SetValue(1);
+                StandardMetrics.ms_v_charge_pilot->SetValue(1);
```

Comment:
I've got a feeling these are boolean metrics. In which case, it would be nice to set them to true rather than 1.

Reply:
That is a good point. I had not thought about that. Makes the code a bit clearer.